pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -456,6 +456,7 @@ func (p *staticPolicy) allocateCPUs(s state.State, numCPUs int, numaAffinity bit
 
 func (p *staticPolicy) guaranteedCPUs(pod *v1.Pod, container *v1.Container) int {
 	if v1qos.GetPodQOS(pod) != v1.PodQOSGuaranteed {
+		klog.V(5).InfoS("CPU Allocation Skipped for pod as it is allocated only for Guaranteed pods", "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name, "qos", v1qos.GetPodQOS(pod))
```

Comment:
I'd compute `qos` once and use it everywhere in the function. Not for performance (I expect negligible gains) but for consistency. This way we use a single coherent value for all the function and we avoid any possible occurrence of TOCTOU issues, even though it's practically impossible because of the code works, structuring the code like that prevents that from the beginning.

I'd rephrase the message like `Exclusive CPU allocation skipped, pod QoS is not guaranteed`

Reply:
Sure, will update.